src/client/Client.cc

```diff
@@ -11605,6 +11605,9 @@ int Client::ll_create(Inode *parent, const char *name, mode_t mode,
 		      int flags, struct stat *attr, Inode **outp, Fh **fhp,
 		      int uid, int gid)
 {
+  if (fhp)
+    *fhp = NULL;
```

Comment:
If it's necessary we can do this unconditionally and clean up comments about "may be" NULL?